basic_flutter/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
```

Comment:
change the SDK so we could all be working with the same SDK and have no build issues